app/views/events/show.html.erb

```diff
@@ -1,69 +1,77 @@
+    </h2>
+</div>
+<!-- End event title -->
+<!-- Left side with image -->
+<div class="container-fluid" style="padding-left:5%; padding-top: 2%; padding-bottom: 1%; width: 30%; float: left;">
   <p>
     <%= image_tag @event.image.url, class: "event-show" %>
   </p>
-<h1><%= @event.name %></h1>
-<br>
-
-<p>
-  <strong>Description:</strong>
-  <%= @event.description %>
-</p>
-
-<p>
-  <strong>Date:</strong>
-  <%= @event.date %>
-</p>
-
-<p>
-  <strong>Location:</strong>
-  <%= @event.location %>
-</p>
-
-<p>
-  <strong>Tickets:</strong>
-  <%= @event.has_tickets %>
-</p>
-
-<p>
-  <strong>Tickets:</strong>
-  $<%= number_with_precision(@event.price, precision: 2) %>
-</p>
-
-<p>
-  <strong>Capacity:</strong>
-  <%= @event.capacity %>
-</p>
-
-<p>
-  <strong>Guests:</strong><br>
-  <% @guests.each do |guest| %>
-    <%= guest.firstname %> <%= guest.lastname %> <% if admin_signed_in? %> | <%= guest.email %> | <%= link_to 'Delete', event_guest_path(@event, guest), method: :delete, data: { confirm: 'Are you sure?' } %>
-    <% end %>
-    <br>
-  <% end %>
-</p>
-
-<p>
-  <strong>Share on Facebook</strong>
-    <%=link_to 'http://www.facebook.com/sharer.php?u='+request.fullpath, :target => :blank do %>
-      <i class="fab fa-facebook-square text-primary"></i>
-    <% end %>
-</p>
+</div>
+<!-- End Left side with image -->
 
-<%= link_to 'Attend', new_event_guest_path(@event), class: "btn btn-primary text-white" if @event.date > Date.current %>
-<%= link_to 'Tasks', event_tasks_url(@event), class: "btn btn-primary text-white"  if admin_signed_in? || volunteer_signed_in? %>
 
-<%= link_to 'Back', events_path, class: "btn btn-secondary text-white" if @event.date > Date.current %>
-<%= link_to 'Back', past_events_path, class: "btn btn-secondary text-white" if @event.date < Date.current %>
+<!-- Center text -->
+<div class="container-fluid" style="padding-right:1%; padding-top: 2%; padding-bottom: 1%; width: 10%; float: left;">
+    <h4><b>Location:</b></h4>
+    <p>
+      <%= @event.location %>
+    </p>
+    <h4><b>Price:</b></h4>
+    <p>
+      $<%= number_with_precision(@event.price, precision: 2) %>
+    </p>
+    <h4><b>Capacity:</b></h4>
+    <p>
+      <%= @event.capacity %>
+    </p>
+    <h4><b>Share:</b></h4>
+    <p>
+        <%=link_to 'http://www.facebook.com/sharer.php?u='+request.fullpath, :target => :blank do %>
+          <i class="fab fa-facebook-square text-primary" style = "font-size:40px"></i>
+        <% end %>
+    </p>
+  </div>
+<!-- End Center text -->
 
+<!-- Right text -->
+  <div class="container-fluid" style="padding-right:5%; padding-top: 2%; padding-bottom: 2%; width: 50%; float: left;">
+      <h4><b>Description:</b></h4>
+        <p>
+          <%= @event.description %>
+        </p>
+        <%= link_to 'Attend', new_event_guest_path(@event), class: "btn btn-primary text-white" if @event.date > Date.current %>
```

Comment:
maybe check my branch and copy in the event.past_event? function to avoid this type of logic in view.